src/storage/queueNumber.js

```diff
@@ -12,6 +13,7 @@ const handleQueueNumber = async () => {
   } catch (e) {
     return {
       statusCode: 400,
+      headers: CORS_ERROR_HEADER,
```

Comment:
try and notice the pattern here
```
try {
    do function stuff
} catch(e) {
    do error stuff
}
```

this pattern is repeated everywhere in this repo, see if you can find a way to extract that pattern out and refactor all handlers in the repo so that there's less of this repetition

Reply:
ok. created a function for try-catch inside ./utils/try-catch.js. cors error header also relocated there since only this file will be using it.